ruby track/exercise14.rb

```diff
@@ -0,0 +1,11 @@
+# Exercise: 14 Reverse Sentence
+
+# Use string methods to reverse the words arrangement in a sentence. Eg: "An apple a day keeps the doctor away" -> "away doctor the keeps day a apple An"
+
+
+def string(s)
+    s=s.split(" ").reverse.join(" ")
```

Comment:
no need to store back the result in s. Just `s.split(" ").reverse.join(" ")` will return the result